java-tic-tac-toe.iml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess in projects I've worked over the last few years we decided to not check in IDE specific files so that developers can have the freedom to use the ide / text editor of their choice.